docs/guide/custom_policies.md

```diff
@@ -122,11 +123,10 @@ class AcmeExampleBundle extends Bundle
 
 To provide support for editing custom policies in the Back Office, you need to implement [`EzSystems\EzPlatformAdminUi\Limitation\LimitationFormMapperInterface`](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/lib/Limitation/LimitationFormMapperInterface.php).
```

Comment:
Some example code here would be really useful. 🙂 It's rather a general statement.

Reply:
@micszo it is too long to do it, i sugest to merge first this PR, to correct the current documentation. After i think it is better to split this chapter, and create an other one for custom limitations, to explain how to create CustomLimitation, CustomLimitationType,  LimitationFormMapperInterface, imitationValueMapperInterface.

Reply:
Yes, internally we also came to the conclusion that this will have to be tackled separately. \cc @dabrt

Reply:
A ticket was created to handle your suggestions to this article:
https://issues.ibexa.co/browse/EZP-32155
Should you have any more comments, you can add them directly there.

Reply:
Thanks!